okhttp/src/main/java/okhttp3/internal/ws/MessageDeflater.kt

```diff
@@ -0,0 +1,31 @@
+package okhttp3.internal.ws
+
+import okio.Buffer
+import okio.ByteString
+import okio.DeflaterSink
+import java.io.Closeable
+import java.io.IOException
+import java.util.zip.Deflater
+
+class MessageDeflater(private val contextTakeover: Boolean) : Closeable {
+  private val sink = Buffer()
+  private val deflater = Deflater(Deflater.BEST_COMPRESSION, true /* omit zlib header */)
```

Comment:
are you sure you want `BEST_COMPRESSION`? Typically  DEFAULT_COMPRESSION is appropriate for network where we need to balance time compressing vs. time transmitting

Reply:
Did not run any benchmarks so far. 
OK, will set it to default compression to be on the safe side. Will raise additional PR if benchmarks show  `BEST_COMPRESSION` fits better.

Reply:
Benchmarking won’t show you much because we’re just deciding how to trade off CPU+memory for network.

Reply:
(benchmarking will tell you that your test environment has lots of network and so any compression at all is wasted)

Reply:
OK, I'm totally fine with `DEFAULT_COMPRESSION`.